controlpanel.cpp

```diff
@@ -50,11 +50,14 @@ ControlPanel::ControlPanel(QWidget *parent, Settings *settings)
     fillOpenModeCombo();
     m_check_lineBreak->setChecked(session.showCtrlCharacters);
     m_check_timestamp->setChecked(session.showTimestamp);
+    m_check_reconnect->setChecked(session.autoReconnect);
 
     connect(m_check_lineBreak, &QCheckBox::toggled,
             [=](bool checked) { emit settingChanged(Settings::ShowCtrlCharacters, checked); });
     connect(m_check_timestamp, &QCheckBox::toggled,
             [=](bool checked) { emit settingChanged(Settings::ShowTimestamp, checked); });
+    connect(m_check_reconnect, &QCheckBox::toggled,
+            [=](bool checked) { emit settingChanged(Settings::AutoReconnect, checked); });
```

Comment:
Without having tested it. I wonder if it would be useful to call 
`m_reconnectTimer.stop();` here too in case checked is set to false.
Otherwise the user would see an error if CuteCom is still trying to reconnect with no device connected but the checkbox being unchecked, wouldn't he?